src/middleware/src/storage/index.js

```diff
@@ -1,22 +1,22 @@
 import { isDefined } from 'kinvey-js-sdk/dist/export';
-import KinveyStorage from 'kinvey-js-sdk/dist/request/src/middleware/src/storage';
-import IndexedDB from './src/indexeddb';
-import WebSQL from './src/websql';
-import { LocalStorage } from './src/webstorage';
+import { Storage as KinveyStorage } from 'kinvey-js-sdk/dist/request/src/middleware/src/storage';
+import { IndexedDBAdapter } from './indexeddb';
```

Comment:
@thomasconner none of the files have moved as part of this commit, so how/why did the relative paths change?